<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_R">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Collect R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Rwarning">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>200</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Collect Rwarning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_photo">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Collect Photo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>501</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_R">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_Rwarning">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_connect">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>640</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_open">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_prev">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_next">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>460</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_localoperations">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>350</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Local Photos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>501</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_quit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>640</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_collectoperations">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Collect Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>501</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <zorder>lb_2</zorder>
   <zorder>lb_1</zorder>
   <zorder>pb_R</zorder>
   <zorder>pb_Rwarning</zorder>
   <zorder>pb_photo</zorder>
   <zorder>lb_pic</zorder>
   <zorder>text_R</zorder>
   <zorder>text_Rwarning</zorder>
   <zorder>pb_connect</zorder>
   <zorder>pb_open</zorder>
   <zorder>pb_prev</zorder>
   <zorder>pb_next</zorder>
   <zorder>pb_delete</zorder>
   <zorder>lb_localoperations</zorder>
   <zorder>pb_quit</zorder>
   <zorder>lb_collectoperations</zorder>
   <zorder>comboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
